rocky/reports/report_types/definitions.py

```diff
@@ -17,6 +17,11 @@ class ReportPlugins(TypedDict):
     optional: set[str]
 
 
+class SubReportPlugins(TypedDict):
+    required: list[str]
+    optional: list[str]
```

Comment:
@ammar92 I have realized that the TypeDict of:
<img width="877" alt="image" src="https://github.com/user-attachments/assets/eb95965c-a108-4c77-80ba-e45660cf2d5b">

is a different one. It is a dict of type Report, buy this is another one. I was confused too. 
It is this one which looks like the plugins:
<img width="436" alt="image" src="https://github.com/user-attachments/assets/6f4f8631-ff44-495e-b4dc-e49d82989f1b">